tests/validation/tests/v3_api/resource/terraform/k3s/master/install_k3s_master.sh

```diff
@@ -20,16 +20,31 @@ fi
 
 if [[ -n "${8}" ]] && [[ "${8}" == *"protect-kernel-defaults"* ]]
 then
-  cat /tmp/cis_masterconfig.yaml >> /etc/rancher/k3s/config.yaml
+  mkdir -p /var/lib/rancher/k3s/server/manifests
+  if [[ "${8}" == *"selinux"* ]]
+  then
+    yum -y install bc
```

Comment:
We should only install this based on OS, not necessarily if the selinux param is passed in config.